### DISCUSSION TITLE
Help changing to new docker-compose.yml and others (Docker Compose Windows 10)

### DISCUSSION THREAD

**Discussion by herculespt** [immich-app/immich#9686] [General]
Hi,

I've been using Immich for several months, and recently I haven't updated the new docker-compose.yml and the .env file.

Today I've been trying to use the recommended docker-compose.yml and I receive an error message: "The  \"DB_DATA_LOCATION\" variable is not set. Defaulting to a blank string" invalid spec: /var/lib/postgresql/data: empty section between colons"

But if i use the "old" docker-compose.yml, it works, I only receive an "obsolete" warning.

I am using Docker Composer in Windows 10, I followed the guide when I start using the Immich about 8 months ago, I update Immich everyday with the command "docker-compose pull && docker-compose up -d", but now I don't know what to do, to use the updated docker-compose.yml and the .env is so different from the one i have also.

attached are the actual files i'am using, If someone can help me, I would appreciate.

![current -  env](https://github.com/immich-app/immich/assets/26182991/874bb4f5-857f-4adf-92f0-aedd11c9c1bb)
![current - hwaccel transcoding yml](https://github.com/immich-app/immich/assets/26182991/b6d6e684-c921-4d36-978e-5bbb22a0b62a)
![current_docker-compose_part1 yml](https://github.com/immich-app/immich/assets/26182991/ccb8231f-0d3e-4734-9697-1803da398cdf)
![current_docker-compose_part2 yml](https://github.com/immich-app/immich/assets/26182991/bbdb3761-b429-453c-9128-302b6bca69b1)

**Comment by herculespt** [comment]
Hi,

I already have done it successfully. But now I am facing a new setup. Meaning, I have no users or admin.

how can I copy all my database settings to the new ./postgree folder created in local disk?

Any help?

**Comment by Zer0x00** [comment]
Please have a look here: https://immich.app/docs/administration/backup-and-restore/

**Reply by herculespt** [reply]
Hi,

Thank you for your help, but I've done it by saving each folder and file in docker compose Postgres mounted volume, and then decompress them in the local hard disk directrory.

**Comment by herculespt** [comment]
it's done. Thank you for no help.

**Reply by stewedrabbit** [reply]
Care to give some pointers on how you solved it? 
Thanks!

**Reply by herculespt** [reply]
Hi. I can do my best to help you out.

I have done it some time ago, and haven't needed it anymore. But from what I have written I make an idea of what I have done.

What do you need?

**Reply by herculespt** [reply]
Well, in Docker, on the left side I have selected Volumes, choosed the one that said "database" or "postgres", then selected one folder and mouse right button, then "Save". I have done it for all folders, and files in the root of that volume. Then I have set a database path in .env, decompressed all those folders inside the choosed folder. And then I just do a pull again.

The docker-compose pull && docker-compose up -d

**Reply by stewedrabbit** [reply]
Thanks for the feedback. I found out my issue was a bit more stupid. I updated the docker-compose.yml, but forgot to update my .env - see release notes for https://github.com/immich-app/immich/releases/tag/v1.102.0

Cheers
